src/strings.sol

```diff
@@ -42,9 +42,9 @@ library strings {
         uint _ptr;
     }
 
-    function memcpy(uint dest, uint src, uint len) private pure {
+    function memcpy(uint dest, uint src, uint len_) private pure {
```

Comment:
nitpick, but maybe just call it `length` to avoid collision

Reply:
`length` is also good. What specific collision will occur?